forge-gui/res/cardsfolder/upcoming/monument_to_perfection.txt

```diff
@@ -0,0 +1,11 @@
+Name:Monument to Perfection
+ManaCost:2
+Types:Artifact
+A:AB$ ChangeZone | Cost$ 3 T | Origin$ Library | Destination$ Hand | ChangeType$ Land.Basic,Land.Locus,Land.Sphere | ChangeTypeDesc$ basic, Sphere, or Locus land card | SpellDescription$ Search your library for a basic, Sphere, or Locus land card, reveal it, put it into your hand, then shuffle.
+A:AB$ Animate | Cost$ 3 | CheckSVar$ CountAll | SVarCompare$ GE9 | Power$ 9 | Toughness$ 9 | Types$ Artifact,Creature,Phyrexian,Construct | RemoveAllAbilities$ True | Defined$ Self | Keywords$ Indestructible & Toxic:9 | SpellDescription$ CARDNAME becomes a 9/9 Phyrexian Construct artifact creature, loses all abilities, and gains indestructible and toxic 9. Activate only if there are nine or more lands with different names among the basic, Sphere, and Locus lands you control.
+SVar:CountAll:Count$DifferentCardNames_Land.YouCtrl+inZoneBattlefield+Basic+!Sphere+!Locus/Plus.CountSpheresAndLocus
+SVar:CountSpheresAndLocus:Count$DifferentCardNames_Land.YouCtrl+inZoneBattlefield+Sphere+!Locus/Plus.CountLocus
+SVar:CountLocus:Count$DifferentCardNames_Land.YouCtrl+inZoneBattlefield+Locus
```

Comment:
Are you thinking this templating means the objects are multiple groups?

I'd say it can still count wrong, e.g. if you have 5 differently named Locus + 5 different Sphere you would end up with 10 but there's a chance it will stay at 5 in the extreme case since you're not checking if the Locus and Spheres have names in common together

Imo the syntax should already support doing it in a single line

Reply:
Yeah, I hadn't thought to much about situations where you somehow change the land types of names of lands you control (since I don't think that's possible atm, but might be in the future).  But I'll think about a better way to implement it, any pointers to cards I should look at ?

Reply:
You can always animate them into creatures and equip with _Spy Kit_ :P

`SVar:CountAll:Count$DifferentCardNames_Land.YouCtrl+inZoneBattlefield+Basic,Land.YouCtrl+inZoneBattlefield+Sphere,Land.YouCtrl+inZoneBattlefield+Locus`